geotrek/common/functions.py

```diff
@@ -1,7 +1,14 @@
 from django.contrib.gis.db.models.functions import GeoFunc
-from django.db.models import FloatField
+from django.db.models import CharField, FloatField, Func
 
 
 class Length(GeoFunc):
     """ ST_Length postgis function """
     output_field = FloatField()
+
+
+def GeometryType(geom):
```

Comment:
can you use a class inherited from GeoFUNC? use attribute function = "GEOMETRY_TYPE" to force